src/main.ts

```diff
@@ -115,7 +118,11 @@ const toggleWindow = () => (window.isVisible() ? window.hide() : showWindow());
 
 const createTray = () => {
   tray = new Tray(path.join(__dirname, 'favicon.png'));
-  tray.on('right-click', toggleWindow);
+  const contextMenu = Menu.buildFromTemplate([
```

Comment:
could this be on right click ? Right now on mac it opens the context menu on left-click also